src/param/param_slash.c

```diff
@@ -30,6 +30,49 @@
 static char queue_buf[PARAM_SERVER_MTU];
 param_queue_t param_queue = { .buffer = queue_buf, .buffer_size = PARAM_SERVER_MTU, .type = PARAM_QUEUE_TYPE_EMPTY, .version = 2 };
 
+static int param_slash_parse_array(char * arg, int node, param_t **param, int *offsets, int *slice_detection) {
+
+	/* Search for the '@' symbol:
+	 * Call strtok twice in order to skip the stuff head of '@' */
+	char * saveptr;
+	char * token;
+	char slice_colon;
+	
+	strtok_r(arg, "[", &saveptr);
+	token = strtok_r(NULL, "[", &saveptr);
+	if (token != NULL) {
+        /* Search for the '[' symbol: */
+		// Searches for the format digit:digit.
+		sscanf(token, "%d%c%d", &offsets[0], &slice_colon, &offsets[1]);
```

Comment:
You might want to be a little defensive in handling the function arguments. Especially the ones which points to data to be "returned". The `offset` pointer and the `slice_detection` could be NULL. So could `param` and `arg`